Dockerfile

```diff
@@ -1,34 +1,57 @@
-#
+### ### ### ### ### ### ### ### ###
 # Builder layer
-#
+
 FROM node:6-alpine as ghost-builder
-RUN npm install --loglevel=error -g ghost-cli
 
-ENV GHOST_VERSION 1.17.1
-ENV GHOST_INSTALL /var/lib/ghost
-ENV GHOST_CONTENT /var/lib/ghost/content
-ENV GHOST_USER node
+RUN \
+    apk update && apk upgrade                       	&& \
+    apk add --no-cache wget unzip ca-certificates   	&& \
+    echo												&& \
+    echo "--- Install ghost-cli --- "; echo         	&& \
+    npm install --loglevel=error -g ghost-cli			;
+
+ENV GHOST_VERSION="1.17.1"							    \
+	GHOST_INSTALL="/var/lib/ghost"					    \
+	GHOST_CONTENT="/var/lib/ghost/content"			    \
+	GHOST_THEMES="/var/lib/ghost/content/themes"	    \
```

Comment:
This variable in the builder is useless right? It is maybe better to move it into the final image?

Reply:
Correct

> these are needed when I override casper theme with my own theme.
Useless in this case.